src/Classic.Avalonia.Theme/Styles/AutoCompleteBox.axaml

```diff
@@ -1,11 +1,12 @@
 <ResourceDictionary xmlns="https://github.com/avaloniaui"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:common="clr-namespace:Classic.CommonControls;assembly=Classic.CommonControls.Avalonia"
                     x:ClassModifier="internal">
   <ControlTheme x:Key="{x:Type AutoCompleteBox}"
                 TargetType="AutoCompleteBox">
-    <Setter Property="Background" Value="{DynamicResource ThemeBackgroundBrush}" />
-    <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderMidBrush}" />
-    <Setter Property="BorderThickness" Value="{DynamicResource ThemeBorderThickness}" />
+    <Setter Property="Background" Value="{DynamicResource {x:Static common:SystemColors.WindowBrushKey}}" />
+    <Setter Property="BorderBrush" Value="{x:Static ClassicBorderDecorator.ClassicBorderBrush}" />
+    <Setter Property="BorderThickness" Value="2" />
```

Comment:
I think we should also reduce the padding, on 640x480 screen, 8 pixels lost just for blank space is too wasteful :D

Reply:
Can't remember why it is there. Will use 1, same as TextBox. 
But generally, there are height issue across different input elements. TextBox is like 19px, and ComboBox is 22px. This need to be addressed in the future.